src/build-config.js

```diff
@@ -0,0 +1,82 @@
+/**
```

Comment:
Can you get a test on this?

Reply:
Specifically-- if a number conversion is undefined -- that is a place that's not obvious we are retaining our current behavior with timeouts